### ISSUE TITLE
How to make slider image doesn't rotate  when setReversed(true) ?

### ISSUE THREAD

**Issue by Farzad-Aghaeizadeh** [cortinico/slidetoact#123]
In some states, I have a problem with setReversed() method, In this case, when setReversed(true) called,  icon image has been rotated 180 degrees and I didn't want that, How can I do that?

It has been written in some issues that in the new version changing the icon image programmatically will be added and this can solve my problem, but when is the new version release date?
is there another solution to do this?

**Comment by cortinico** [comment]
> It has been written in some issues that in the new version changing the icon image programmatically will be added and this can solve my problem, but when is the new version release date?
> is there another solution to do this?

Correct, the feature merged in #119 will allow you to override the image at runtime with whatever you want.

Given that the last release was done in March, I can imagine releasing another version soon (like this week). In the meanwhile you can try the version on top of master with JitPack and let me know if that unblocks you: https://jitpack.io/#cortinico/slidetoact/

```
// root build.gradle
allprojects {
    repositories {
        // ...
        maven { url 'https://jitpack.io' }
    }
}

// local build.gradle
dependencies {
	implementation 'com.github.cortinico:slidetoact:master-SNAPSHOT'
}
```

**Comment by Farzad-Aghaeizadeh** [comment]
Dear Cortinico, thank you.
It helps me, a very useful comment 👌.

**Comment by cortinico** [comment]
This is now included in version 0.9.0 that was release just now: 
https://github.com/cortinico/slidetoact/releases/tag/v0.9.0
